cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -29,7 +29,7 @@
 ]
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="function")
```

Comment:
Optional: no need to pass `scope="function"` since `pytest.fixture` is function scoped by default.

Reply:
yup... as appliance only inherited.

Reply:
@john-dupuy thanks!